src/main/java/racing/domain/Racing.java

```diff
@@ -0,0 +1,13 @@
+package racing.domain;
+
+import java.util.Arrays;
+
+public class Racing {
+
+    public CarList oneCycle(CarList carList) {
+        Arrays.stream(carList.getCars())
+                .forEach(car -> car.move(new RandomlyMove()));
+        return carList;
```

Comment:
일급 컬렉션 객체를 만들어주셨으니, getter를 사용하지 않고 메시지를 전송하는건 어떨까요?

```suggestion
        carList.move(new RandomlyMove())
        return carList;
```